stingray/io.py

```diff
@@ -38,6 +39,63 @@
 logger = setup_logger()
 
 
+def read_rmf(rmf_file=None):
+    """Load RMF info.
+
+    .. note:: Preliminary: only EBOUNDS are read.
+
+    Parameters
+    ----------
+    rmf_file : str
+        The rmf file used to read the calibration. If None or not specified,
+        the one given by default_nustar_rmf() is used.
+
+    Returns
+    -------
+    pis : array-like
+        the PI channels
+    e_mins : array-like
+        the lower energy bound of each PI channel
+    e_maxs : array-like
+        the upper energy bound of each PI channel
+    """
+
+    lchdulist = pf.open(rmf_file, checksum=True)
```

Comment:
Otherwise the response stuff lgtm

Reply:
@matteolucchini1 good point. I often do the explicit open/close because Astropy has a nonstandard behavior when associating the `with` statement with the default memory mapping of FITS files (see discussion here: https://github.com/astropy/astropy/issues/7404). By forcing memmap=False, this syntax should be safe.